src/webgl/p5.Framebuffer.js

```diff
@@ -894,17 +882,51 @@ class Framebuffer {
   }
 
   /**
-   * After having previously called
-   * <a href="#/p5.Framebuffer/begin">begin()</a>, this method stops drawing
-   * functions from going to the framebuffer's texture, allowing them to go
-   * right to the canvas again. After this, one can read from the framebuffer's
-   * texture.
+   * When setting up a framebuffer to be written to, which WebGL framebuffer
```

Comment:
Can you clarify this sentence? I'm having a hard time understanding what is meant by this first sentence